thirdparty/README.md

```diff
@@ -630,13 +630,13 @@ Files extracted from upstream source:
 For 2D in `rvo2_2d` folder
 
 - Upstream: https://github.com/snape/RVO2
-- Version: git (f7c5380235f6c9ac8d19cbf71fc94e2d4758b0a3, 2021)
+- Version: git (a7a9575b24f6972f5edd0790784860406663c08f, 2023)
```

Comment:
That commit is actually from 2022, not 2023.

There's a few commits in 2023 though, last one yesterday, might be worth syncing with those.

Reply:
Probably because the latest release in the repo is from 2016